.github/workflows/build.yml

```diff
@@ -91,3 +95,6 @@ jobs:
             path: |
               ${{ github.workspace }}/src/**/Release/*.nupkg
 
+        - name: List vulnerable packages
```

Comment:
We've removed building samples and other bits from CI to speed things up. This sounds like a good opportunity to parallelize CI with an extra workflow or job. Is that an option?

(Assuming it adds a minute as you mentioned in a previous chat)

Reply:
@bruno-garcia @mattjohnsonpint i changed it to be scheduled once a day. thoughts?